src/main/resources/public/ts/utils/workspace-entcore.utils.ts

```diff
@@ -53,14 +53,13 @@ export class WorkspaceEntcoreUtils {
     static updateWorkspaceDocuments(folder: any | models.Element): void {
         if (folder && folder instanceof models.Element) {
             if ("tree" in folder) {
-                WorkspaceEntcoreUtils.workspaceScope()['onTreeInit'](() =>
-                    WorkspaceEntcoreUtils.workspaceScope()['setCurrentTree'](folder['tree']['filter'])
-                )
-            } else if (folder._id && folder.eType === "folder") {
-                WorkspaceEntcoreUtils.workspaceScope()['onTreeInit'](() =>
-                    WorkspaceEntcoreUtils.workspaceScope()['openFolderById'](folder._id)
-                )
+                folder.eType = 'folder';
```

Comment:
We have to add the atribute eType to the root folder because the root folder does not have it by default and without this attibute, this folder is not included in the tree update.